### PR TITLE
chore: use prettier for local formatting

### PR DESCRIPTION
**Pull request by JustinBeckwith**
This is the first step towards a prettier migration.  Adding the dependency, and using it to format our source only as a first step.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,7 +24,7 @@
     "codecov": "nyc report --reporter=json && codecov -f coverage/*.json",
     "compile": "tsc -p .",
     "format-check": "./bin/format-check.sh",
```
Comment (ofrobots) [inline package.json]: Can this be just `prettier --list-different glob` now?

File: src/clean.ts [hunk 2]
```diff
@@ -32,8 +32,9 @@ export async function clean(options: Options): Promise<boolean> {
     const outDir = tsconfig.compilerOptions.outDir;
     if (outDir === '.') {
       options.logger.error(
-          `${chalk.red('ERROR:')} ${chalk.gray('compilerOptions.outDir')} ` +
-          `cannot use the value '.'.  That would delete all of our sources.`);
+        `${chalk.red('ERROR:')} ${chalk.gray('compilerOptions.outDir')} ` +
+          `cannot use the value '.'.  That would delete all of our sources.`
```
Comment (ofrobots) [inline src/clean.ts]: meh.

I guess I will get used to it.

File: src/cli.ts [hunk 3]
```diff
@@ -37,8 +37,11 @@ export interface Options {
   logger: Logger;
 }
 
-export type VerbFilesFunction =
-    (options: Options, files: string[], fix?: boolean) => Promise<boolean>;
+export type VerbFilesFunction = (
+  options: Options,
+  files: string[],
+  fix?: boolean
+) => Promise<boolean>;
```
Comment (ofrobots) [inline src/cli.ts]: sigh.

I guess I will get used to it.

File: src/format.ts [hunk 4]
```diff
@@ -37,8 +37,10 @@ export interface Replacement {
 export const clangFormat = require('clang-format');
 
 const BASE_ARGS_FILE = ['-style=file'];
-const BASE_ARGS_INLINE =
-    ['-style', '{Language: JavaScript, BasedOnStyle: Google, ColumnLimit: 80}'];
+const BASE_ARGS_INLINE = [
+  '-style',
+  '{Language: JavaScript, BasedOnStyle: Google, ColumnLimit: 80}',
+];
```
Comment (ofrobots) [inline src/format.ts]: whatevs.

File: src/format.ts [hunk 5]
```diff
@@ -47,18 +49,22 @@ const BASE_ARGS_INLINE =
...
   // inline argument.
-  const baseClangFormatArgs =
-      fs.existsSync(path.join(options.targetRootDir, '.clang-format')) ?
-      BASE_ARGS_FILE :
-      BASE_ARGS_INLINE;
+  const baseClangFormatArgs = fs.existsSync(
+    path.join(options.targetRootDir, '.clang-format')
+  )
+    ? BASE_ARGS_FILE
+    : BASE_ARGS_INLINE;
```
Comment (ofrobots) [inline src/format.ts]: :crying_cat_face:

File: package.json (hunk 1 above)
Comment (JustinBeckwith) [reply package.json]: Yes it can!  Good call out 👍

**Pull request merged**